torch/nn/modules/normalization.py

```diff
@@ -133,12 +134,16 @@ class LayerNorm(Module):
         >>> m = nn.LayerNorm(input.size()[1:])
         >>> # Without Learnable Parameters
         >>> m = nn.LayerNorm(input.size()[1:], elementwise_affine=False)
-        >>> # Normalize over last two dimensions
-        >>> m = nn.LayerNorm([10, 10])
         >>> # Normalize over last dimension of size 10
         >>> m = nn.LayerNorm(10)
+        >>> # Normalize over last three dimensions
```

Comment:
What do you think about a small tweak to this example to use the standard CV dimensions? Something like:

```python
N, C, H, W = 20, 5, 10, 10
input = torch.randn(N, C, H, W)
```

with this comment changed to:
`Normalize over the last three dimensions (i.e. the channel and spatial dimensions)`

Then it's more clear how to achieve the standard normalization shown in the `LayerNorm` block diagram. I like how you described in the PR description how to achieve this, and I think it'd be valuable to have in the docs somehow. Ooh is it possible to add the image as well?